main/politeness_logi.R

```diff
@@ -15,14 +15,17 @@ system2("src/convo_politeness",
         input = format_csv(dat))
 
 # read python's output from file
-dat <- read.csv("politeness_features.csv")
-names(dat)
+dat <- read.csv("politeness_features.csv", stringsAsFactors=FALSE)
+dat[is.na(dat)] <- 0
+summary(dat$Indirect_.btw.)
+sapply(dat, class)
 m_pol <- glm(label ~
-              log(1 + HASHEDGE)
+              log(1+length)
+            + log(1 + HASHEDGE)
             + (Please > 0)
             + (Please_start > 0)
             + (Indirect_.btw. > 0)
-            + Hedges
+            #+ Hedges
```

Comment:
add back in?

Reply:
I think it should be removed. According to their paper, `HASHEDGE` is a measure, but they didn't mention `Hedges`. Also, looking at the words marked for the category (Hedges), they are not hedge words. They seem to be words that appear around hedge words.

Reply:
Okey dokey. Probably add a comment to the code then.